scripts/build/build/target.py

```diff
@@ -95,6 +95,24 @@ def Accept(self, full_input: str):
 
         return True
 
+    def ToDict(self):
+        """Converts a TargetPart into a dictionary
+        """
+
+        result: Dict[str, str] = {}
+        result['name'] = self.name
+        bld_args_keys = list(self.build_arguments.keys())
```

Comment:
build_arguments seems to already be a dict, so how about:

```python
for key, value in self.build_arguments.items():
    result[key] = str(value)
```

I also wonder if it would be maybe safer to put this in `result['build_arguments']` since there seems no guarantee that build_arguments does not contain something named `name` or `only_if_re` or such.

Reply:
Oops. This code is from when I was processing the TargetPart obects from BuildTarget and needed to get access to the underlying data.

Reply:
Yes, makes sense, that's safer to put in it build_arguments.